### DISCUSSION TITLE
Modular post-processing

### DISCUSSION THREAD

**Discussion by seabbs** [epinowcast/epinowcast#124] [Ideas]
We now have a modular system of pre-processing and it would be nice if we could do the same for stan code and postprocessing. Of these modular stan code is the hardest so likely best to focus on modular post-processing. The first step towards this is reviewing the current code with a eye to making it modular and making methods downstream of `epinowcast` infer what to extract (and what is available to extract) from the `epinowcast` object.

**Comment by adrian-lison** [comment]
This discussion continues in our discourse community [here](https://community.epinowcast.org/t/modular-post-processing/38).
